include/drivers/ca/ca_bscan.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2021 Wavious LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _CA_BSCAN_H_
+#define _CA_BSCAN_H_
+
+#include <stdbool.h>
+#include "ca_reg.h"
+
+/*******************************************************************************
+**                            CA
+*******************************************************************************/
+/**
+ * @brief   CA DQ BSCAN Get Status Register Interface
+ *
+ * @details Gets the CA slice BSCAN status.
```

Comment:
Is this supposed to say DQ slice BSCAN status?

Reply:
No DQ will be in the next commit.